.fixtures.yml

```diff
@@ -3,3 +3,6 @@ fixtures:
     stdlib:
       repo: 'https://github.com/puppetlabs/puppetlabs-stdlib.git'
       ref: '4.22.0'
+    concat:
+      repo: 'https://github.com/puppetlabs/puppetlabs-concat.git'
+      ref: '4.0.0'
```

Comment:
v6.2.0 is the latest. Note this also needs added to `.fixtures-latest.yml`

Reply:
I thought this file was for  testing the lower bound of what's in metadata.json?  I will update `.fixtures-latest.yml`.

Reply:
yeah, but since we're doing a new version, we should not be tied to supported really old versions of dependencies. note, i need to bump stdlib as well.

Reply:
I've bumped minimum to 6.0.0